redhat-developer/developers.redhat.com
# [MWES-2846] Update Drupal Docker Image to RHEL 7 using SCL dependencies

This PR updates our Drupal Docker image to build from the Managed Platform base image. The Managed Platform base image provides SCL versions of most of our dependencies and also provides Apache 2.4 for Drupal.

This PR targets `stage` branch in the first instance so that we can ensure things work as expected once this has been merged in.

### JIRA Issue Link
* https://issues.jboss.org/browse/MWES-2846

### Verification Process

* Build should go green
* Login to Drupal and make sure there aren't any suspicious looking errors
